From bbf3f2787e8e911149c593206905fb1874e71b6b Mon Sep 17 00:00:00 2001
From: Chenghai Huang <huangchenghai2@huawei.com>
Date: Wed, 10 Jun 2026 09:34:37 +0800
Subject: [PATCH 02452/15333] MAINTAINERS: update hisilicon zip driver
 maintainer

Add Chenghai Huang as the maintainer of the hisilicon zip driver,
replacing Yang Shen.

Signed-off-by: Chenghai Huang<huangchenghai2@huawei.com>
Reviewed-by: Longfang Liu <liulongfang@huawei.com>
Acked-by: Zhou Wang <wangzhou1@hisilicon.com>
Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
---
 MAINTAINERS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index 15011f5752a9..d1dbf2f07104 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -11837,7 +11837,7 @@ W:	http://www.hisilicon.com
 F:	drivers/spi/spi-hisi-sfc-v3xx.c
 
 HISILICON ZIP Controller DRIVER
-M:	Yang Shen <shenyang39@huawei.com>
+M:	Chenghai Huang<huangchenghai2@huawei.com>
 M:	Zhou Wang <wangzhou1@hisilicon.com>
 L:	linux-crypto@vger.kernel.org
 S:	Maintained
-- 
2.55.0

